care-o-bot/105dc16/105dc16.bug

```diff
@@ -43,7 +44,7 @@ fix:
     - cob_control/cob_twist_controller/CMakeLists.txt
   languages:
     - CMake
-  time: 2014-11-21 (12:57)
+  time: 2014-11-14T08:16:00Z
```

Comment:
that's a big difference? Does this have to do with the fact it wasn't released?

Reply:
the package was not released during the issue creation but this is the date I got from the script.